build.gradle

```diff
@@ -257,6 +257,8 @@ dependencies {
     rewrite("org.openrewrite.recipe:rewrite-logging-frameworks")
     rewrite("org.openrewrite.recipe:rewrite-testing-frameworks")
     rewrite("org.openrewrite.recipe:rewrite-migrate-java")
+
+    implementation 'com.github.Dansoftowner:jSystemThemeDetector:3.6'
```

Comment:
Please try to keep the order in `build.gradle` --> move it up to the other implementation calls.